lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -130,6 +132,23 @@ def docs
 
     private
 
+    def capture3(command)
```

Comment:
While reading the code initially the unscoped `capture3` confused me. I don't think it should have the same name. Perhaps `run_command` would be clearer?